modules/remotecontrol/Receiver.js

```diff
@@ -200,6 +202,7 @@ export default class Receiver extends RemoteControlParticipant {
      * @returns {void}
      */
     deny(userId: string) {
+        this.emit('active-changed', false);
```

Comment:
It seems worthwhile to me to put this "active-changed" string in a constant, at least to be consistent with how most conference and UI events are in constants.